plans/2026-08-14-browser-annotation-hold-to-view-original.md

```
@@ -0,0 +1,217 @@
+
+```js
+H = l
+  ? <FormattedMessage id="thread.browser.tweaks.originalTitle" defaultMessage="Original • {url}" …/>
+  : <FormattedMessage id="thread.browser.tweaks.title" defaultMessage="Annotating • {url}" …/>
+```
+
+### 2.2 图标（Codex 原始 SVG path，20×20，fill currentColor）
+
+普通「眼睛」（`AVo`）：
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)